lib/device_api/android.rb

```diff
@@ -32,8 +32,9 @@ def self.device(serial)
     end
 
     # Return the device type used in determining which Device Object to create
-    def self.get_device_type(serial)
-      return :default if Device.new(serial: serial).manufacturer.nil?
+    def self.get_device_type(options)
+      return :default if options.values.first == 'unauthorized'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.